etc/config/c++.amazon.properties

```diff
@@ -465,15 +465,19 @@ compiler.arm921.supportsBinary=false
 
 ###############################
 # GCC for Kalray
-group.kalray.compilers=k1cg741:k1cg750
+group.kalray.compilers=kvxg750:k1cg741:k1cg750
 group.kalray.groupName=Kalray MPPA GCC
 group.kalray.isSemVer=true
+compiler.k1cg750.exe=/opt/compiler-explorer/kvx/gcc-7.5.0/kvx-unknown-elf/bin/kvx-unknown-elf-g++
```

Comment:
This compiler isn't referenced above (`k1cg750` vs `kvxg750`) -- which would you prefer? I can update the PR once yo ulet me know which is the one you want to keep :) I presume the `kvxg*` ?

Reply:
Sorry about that, fixed it !